tsconfig.json

```diff
@@ -14,6 +14,12 @@
 		"moduleResolution": "node",
 		"noImplicitAny": true,
 		"noImplicitThis": true,
+		"plugins": [
```

Comment:
what's this?

Reply:
It loads a tslint plugin so linting errors come through in the language services. We're using it for Intern 4 and makes the development experience quite nice.

Reply:
Okay. We should probably put an issue in meta if we want to add that to the Dojo 2 packages.

Reply:
rather than include it in core as part of the intern 4 conversion.